apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -36,6 +36,7 @@ class LessonEditor extends Component {
     initialObjectives: PropTypes.arrayOf(PropTypes.object).isRequired,
     initialLessonData: PropTypes.object,
     unitInfo: PropTypes.object,
+    hasRubric: PropTypes.bool,
```

Comment:
I think this works but since there are no exisiting lessons with rubrics, I could only test it by passing true and false in for this prop.